framework/sources/HFCustomEncoding.m

```diff
@@ -101,8 +105,15 @@ - (NSString *)stringFromBytes:(const unsigned char *)bytes length:(NSUInteger)le
 }
 
 - (NSData *)dataFromString:(NSString *)string {
-    NSLog(@"UNIMPLEMENTED: %@", string);
-    return nil;
+    NSMutableData *bytes = [NSMutableData dataWithLength: 0];
```

Comment:
I think this could be shortened to `NSMutableData.data`

Reply:
Thanks!